04_naive_bayes/04_naive_bayes.jl

```diff
@@ -262,40 +261,26 @@ As you can see below, the model (at least under this simple metric) is performin
 # ╔═╡ aa9f7ea4-2850-11eb-33e2-ade40fd0a360
 spam_filter_accurracy(x_test, y_test, spam_filter, 1)
 
-# ╔═╡ 55955e22-28fc-11eb-0153-83f34561665b
-# to create the emails corpus based on the frequencies of each word
-
-function create_spam_and_ham_corpus(model::BayesSpamFilter, df)
-	
-	n_spam_mails = sum(df.Prediction)
-	n_ham_mails = size(df)[1] - n_spam_mails
-	
-	ham_corpus = Array{String}(undef, n_ham_mails)
-	spam_corpus = Array{String}(undef, n_spam_mails)
-	
-	for i in 1:n_spam_mails
-		spam_corpus[i] = string([repeat(string(word, " "), 												model.words_count_spam[word]) for word in vocabulary])
-	end
-	
-	for i in 1:n_ham_mails
-		ham_corpus[i] = string([repeat(string(word, " "), 
-					model.words_count_ham[word]) for word in vocabulary])
-	end
-	return ham_corpus, spam_corpus
-end	
-
-# ╔═╡ e8ea2bd0-2903-11eb-176f-a527049a3968
-# ham_corpus, spam_corpus = create_spam_and_ham_corpus(spam_filter, raw_df)
-
-# ╔═╡ 7fb08fa2-29b9-11eb-39d5-f74320ac9609
+# ╔═╡ 3ce2070a-7156-11eb-1204-3d1850c7abee
 md"
-#### To do:
-* WordClouds
-* Confusion matrix
+### Summary
+In this chapter, we have used a naive-bayes approach to build a simple email spam filter. 
```

Comment:
don't use the past tense to describe what this chapter will talk about. The past tense would be correct if the summary was at the end of the chapter.
e.g.: "We will use a naive-bayes approach"

Reply:
Discussing the origin of the word spam is a mere curiosity, it is not one of the core contents of this chapter, so it doesn't belong in the summary